exp/go-templates/main.go

```diff
@@ -7,6 +7,7 @@ import (
 	"strings"
 	"syscall/js"
 
+	"github.com/Masterminds/sprig"
```

Comment:
Noting the size increase in the binary, from CircleCI build logs, latest `main` [1] build size for the WASM binary was 5.4 MB:
```
Uploading /home/circleci/project/app/site/public/js/exp/wasm/go-templates.7740e65c73869159109a2b45a9bb5f62e55d89b251cd60625ef0c16b4de4b4aa.wasm (5.4 MB): DONE
```
On this branch [2] it is now 7.6 MB
```
Uploading /home/circleci/project/app/site/public/js/exp/wasm/go-templates.c9f3024b030f653b4f47b589f6bf2cdc17125f0fab2e268e696dd33de39e9513.wasm (7.6 MB): DONE
```

[1] https://app.circleci.com/pipelines/github/kujenga/website/300/workflows/ba320640-c818-448d-bd0a-d12058787526/jobs/660
[2] https://app.circleci.com/pipelines/github/kujenga/website/304/workflows/2bf0a927-7459-4374-9177-3e2151da3991/jobs/672

Reply:
After compression, that translates to:
- 2.28MB on this branch
- 1.62MB on `main`